test/node/get-role.test.ts

```diff
@@ -476,20 +476,28 @@ describe('getRole', () => {
     ['textarea', { given: [{ tagName: 'textarea' }], want: 'textbox' }],
     ['thead', { given: [{ tagName: 'thead' }], want: 'rowgroup' }],
     ['tfoot', { given: [{ tagName: 'tfoot' }], want: 'rowgroup' }],
-    ['th', { given: [{ tagName: 'th' }], want: 'cell' }],
+    ['th', { given: [{ tagName: 'th' }], want: 'rowheader' }],
```

Comment:
This was a bit of a coin flip deciding between `rowheader` and `columnheader` when we:

1. don’t have access to the DOM (and therefore can’t determine siblings)
2. aren’t explicitly told this is NOT in a table
3. aren’t given any information as to what part of the table it’s in

Because the assumption is that every row is inside a `<tbody>` in most browsers by default, and we assume [`<th>` elements in `<tbody>` are rowheaders, even when that may not be true](https://github.com/drwpow/html-aria/issues/25#issuecomment-2824387248), it seemed like statistically the more-likely role. But again, don’t feel strongly, is basically a coin flip.